### PR TITLE
TBR value as percentages

### PR DESCRIPTION
**Pull request by samspycher**
This is a 1:1 copy of the function MedtronicPumpPlugin.setTempBasalPercent into OmnipodPumpPlugin. 

It is tested with multiple temp basal rates featuring both absolute and percentage values, and seems to work fine.

Fixes AAPS-Omnipod/AndroidAPS#10

### CODE REVIEW

File: omnipod/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/OmnipodPumpPlugin.java [hunk 1]
```diff
@@ -895,8 +895,14 @@ public void finishHandshaking() {
...
     @NonNull @Override public PumpEnactResult setTempBasalPercent(Integer percent, Integer
             durationInMinutes, Profile profile, boolean enforceNew) {
-        aapsLogger.debug(LTag.PUMP, "setTempBasalPercent [OmnipodPumpPlugin] - Not implemented.");
-        return getOperationNotSupportedWithCustomText(info.nightscout.androidaps.core.R.string.pump_operation_not_supported_by_pump_driver);
+        if (percent == 0) {
+            return setTempBasalAbsolute(0.0d, durationInMinutes, profile, enforceNew);
+        } else {
+            double absoluteValue = profile.getBasal() * (percent / 100.0d);
+            absoluteValue = pumpDescription.pumpType.determineCorrectBasalSize(absoluteValue);
+            aapsLogger.warn(LTag.PUMP, "setTempBasalPercent [MedtronicPumpPlugin] - You are trying to use setTempBasalPercent with percent other then 0% (" + percent + "). This will start setTempBasalAbsolute, with calculated value (" + absoluteValue + "). Result might not be 100% correct.");
```
Comment (bartsopers) [inline omnipod/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/OmnipodPumpPlugin.java]: I'd change this to OmnipodPumpPlugin ;-)

**Review by bartsopers — COMMENTED** [review]
I'm not sure whether we have unit tests for OmnipodPumpPlugin yet, but I would prefer to add a test for this new method, as we transform the input that is passed to setTempBasalAbsolute(). We should verify that we do that correctly.

I think this should be fairly easy to achieve by constructing OmnipodPumoPlugin with a mocked AapsOmnipodManager and verifying that setTempBasal() has been called with the expected arguments when you call OmnipodPumpPlugin.setTbrPercentage()

Let me know if you need any help!

File: omnipod/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/OmnipodPumpPlugin.java (hunk 1 above)
Comment (samspycher) [reply omnipod/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/OmnipodPumpPlugin.java]: Super, thanks for the feedback bartsopers! Yes, I was expecting a conversation about testing :-)  –  tests coming up.

**Comment by samspycher** [comment]
Unit tests added, was a bit unsure of the reasonable scope of input values for this test => happy for feedback.

**Review by bartsopers — APPROVED** [review]

**Pull request merged**